.env.example

```
@@ -11,9 +10,26 @@ DB_NAME=
 DB_SYNCHRONIZE=true
 DB_SCHEMA:
 
-#list of permissions.
-SCOPES= []
-EGO_URL: "https://ego.url"
+# ----- Auth -----
+
+# client id and secret as present is keycloak or ego for this service
+CLIENT_ID=
+CLIENT_SECRET=
+
+# scopes as defined in EGO or KEYCLOAK eg. DICTIONARY.WRITE (For Keycloak DICTIONAY is the rsname i.e resource name and WRITE is the scope)
+# SCOPES property can be empty or absent for AUTH_STRATEGY = NONE.
+SCOPES=[]
```

To be more clear that these scopes provide write permissions, can we rename to:
```suggestion
AUTH_WRITE_SCOPES=[]
```